From 46313695addc1564e91241d7b13d378bf5b6cc8e Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:27:54 -0400
Subject: [alsa] usb-audio: Add quirk for Logitech Webcam C500

Message-id: <140972823573-232-patchset-jkysela@redhat.com>
Patchwork-id: 90932
O-Subject: [RHEL 7.1 PATCH 231/442] ALSA: usb-audio: Add quirk for Logitech Webcam C500
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Logitech C500 (046d:0807) needs the same workaround like other
Logitech Webcams.

Cc: <stable@vger.kernel.org>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: e805ca8b0a9b6c91099c0eaa4b160a1196a4ae25)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/usb/mixer.c b/sound/usb/mixer.c
index fcd3ab9..d40a285 100644
--- a/sound/usb/mixer.c
+++ b/sound/usb/mixer.c
@@ -888,6 +888,7 @@ static void volume_control_quirks(struct usb_mixer_elem_info *cval,
   }
   break;
 
+ case USB_ID(0x046d, 0x0807): /* Logitech Webcam C500 */
  case USB_ID(0x046d, 0x0808):
  case USB_ID(0x046d, 0x0809):
  case USB_ID(0x046d, 0x081b): /* HD Webcam c310 */
-- 
1.7.1